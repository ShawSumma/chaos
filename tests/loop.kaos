list a = [
    1,
    2,
    3
]

20 times do:
    print a
end

num b = 32
print b
