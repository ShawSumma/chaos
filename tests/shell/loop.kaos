num a = 5
a
print a

5 times do
    print a
end
