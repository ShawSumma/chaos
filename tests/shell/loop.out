[0;90m###[0m [0;90m###[0m 5
[0;90m###[0m 5
[0;90m###[0m [0;90m###[0m [0;90m...[0m [0;90m...[0m 5
5
5
5
5
[0;90m###[0m 